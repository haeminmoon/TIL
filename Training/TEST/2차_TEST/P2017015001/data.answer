Case #1
1
Case #2
5
Case #3
9
Case #4
11
Case #5
10
Case #6
7
Case #7
12
Case #8
14
Case #9
9
Case #10
16
Case #11
13
Case #12
15
Case #13
15
Case #14
12
Case #15
17
Case #16
19
Case #17
18
Case #18
32
Case #19
23
Case #20
34
Case #21
25
Case #22
17
Case #23
20
Case #24
21
Case #25
23
Case #26
19
Case #27
14
Case #28
9
Case #29
33
Case #30
23
Case #31
10
Case #32
25
Case #33
19
Case #34
24
Case #35
6
Case #36
29
Case #37
17
Case #38
23
Case #39
25
Case #40
5
Case #41
31
Case #42
28
Case #43
31
Case #44
24
Case #45
24
Case #46
43
Case #47
15
Case #48
43
Case #49
27
Case #50
42
Case #51
1
Case #52
27
Case #53
26
Case #54
31
Case #55
25
Case #56
25
Case #57
41
Case #58
25
Case #59
35
Case #60
22
Case #61
23
Case #62
37
Case #63
32
Case #64
38
Case #65
34
Case #66
10
Case #67
36
Case #68
41
Case #69
13
Case #70
36
Case #71
30
Case #72
9
Case #73
31
Case #74
39
Case #75
50
Case #76
38
Case #77
54
Case #78
42
Case #79
19
Case #80
31
Case #81
34
Case #82
51
Case #83
21
Case #84
42
Case #85
46
Case #86
48
Case #87
24
Case #88
52
Case #89
44
Case #90
50
Case #91
30
Case #92
37
Case #93
42
Case #94
42
Case #95
30
Case #96
43
Case #97
24
Case #98
26
Case #99
37
Case #100
19
