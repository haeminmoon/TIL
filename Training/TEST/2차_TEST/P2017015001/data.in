100
19 20 221
12 5 56
1 4 1
17 3 19
13 1 1
18 15 91
12 7 8
8 15 91
16 7 57
20 9 19
8 1 5
20 18 289
6 11 45
14 15 16
16 7 71
7 14 29
17 4 21
1 20 1
2 11 1
1 6 1
15 15 46
12 1 5
1 2 1
12 13 14
2 4 5
4 17 52
12 15 151
5 12 49
13 12 97
10 14 15
4 6 13
19 6 37
15 17 35
10 13 66
18 5 36
16 4 9
3 10 21
16 8 97
16 5 11
9 11 23
2 3 1
1 12 1
15 2 5
7 8 17
14 19 1
10 17 1
16 6 73
12 5 11
12 12 121
5 3 1
16 11 100
15 5 36
20 20 261
6 17 1
8 14 29
5 3 4
3 12 1
2 7 8
16 1 2
19 13 105
15 20 101
2 11 1
2 11 1
8 19 20
8 5 36
2 18 1
7 9 28
20 14 253
20 12 157
17 17 86
4 15 1
11 14 85
11 9 91
6 12 37
6 2 7
1 11 1
5 10 31
10 9 1
12 1 9
19 9 127
2 14 15
20 18 37
7 9 10
16 8 121
3 4 9
9 19 134
8 4 9
9 15 106
3 12 25
8 19 134
5 13 53
2 8 9
5 9 19
4 2 3
4 12 13
3 7 1
2 11 12
2 4 5
16 11 34
16 11 122
