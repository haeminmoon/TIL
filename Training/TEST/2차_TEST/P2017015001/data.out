Case #1
2
Case #2
2
Case #3
4
Case #4
3
Case #5
13
Case #6
3
Case #7
11
Case #8
2
Case #9
2
Case #10
10
Case #11
2
Case #12
2
Case #13
2
Case #14
14
Case #15
2
Case #16
4
Case #17
4
Case #18
20
Case #19
22
Case #20
6
Case #21
5
Case #22
3
Case #23
2
Case #24
12
Case #25
2
Case #26
2
Case #27
2
Case #28
2
Case #29
2
Case #30
10
Case #31
2
Case #32
4
Case #33
8
Case #34
2
Case #35
3
Case #36
8
Case #37
2
Case #38
2
Case #39
8
Case #40
5
Case #41
6
Case #42
12
Case #43
6
Case #44
4
Case #45
266
Case #46
170
Case #47
2
Case #48
6
Case #49
2
Case #50
15
Case #51
2
Case #52
3
Case #53
2
Case #54
102
Case #55
4
Case #56
4
Case #57
36
Case #58
2
Case #59
8
Case #60
3
Case #61
3
Case #62
22
Case #63
22
Case #64
8
Case #65
2
Case #66
36
Case #67
3
Case #68
2
Case #69
2
Case #70
4
Case #71
60
Case #72
2
Case #73
2
Case #74
2
Case #75
2
Case #76
11
Case #77
2
Case #78
90
Case #79
2
Case #80
2
Case #81
2
Case #82
10
Case #83
7
Case #84
2
Case #85
2
Case #86
2
Case #87
4
Case #88
2
Case #89
2
Case #90
2
Case #91
2
Case #92
2
Case #93
3
Case #94
3
Case #95
4
Case #96
21
Case #97
2
Case #98
2
Case #99
6
Case #100
2
